src/debugger/launcher.ts

```diff
@@ -37,6 +38,8 @@ export class Launcher {
                     let extensionMessageSender = new em.ExtensionMessageSender();
                     return Q({})
                         .then(() => {
+                            generator.step("checkPlatformCompatibility");
+                            TargetPlatformHelper.checkTargetPlatformSupport(runOptions.platform);
```

Comment:
No. The purpose of this is to block known targets on known incompatible host OS machines.